atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -135,6 +136,10 @@ def set_help(self, extra, worker_metadatas):
                         # They are all None
                         extra['image']['help'] = None
 
+    def set_brew_versions(self, extra, worker_metadatas):
+        build_id, brew_version = self.workflow.postbuild_results.get(PulpPullPlugin.key)
```

Comment:
How does it get added to the worker_metadatas?
Does it happen automatically, or do I have to add code to store_metadata_in_osv3.py similar to the code for the help file?